src/scrabble.js

```diff
@@ -1,5 +1,51 @@
-function scrabble() {
-  // write code here
+const scrabble = (myWord) => {
+  const wordValues = {
+    a: 1,
+    e: 1,
+    i: 1,
+    o: 1,
+    u: 1,
+    l: 1,
+    n: 1,
+    r: 1,
+    s: 1,
+    t: 1,
+    d: 2,
+    g: 2,
+    b: 3,
+    c: 3,
+    m: 3,
+    p: 3,
+    f: 4,
+    h: 4,
+    v: 4,
+    w: 4,
+    y: 4,
+    k: 5,
+    j: 8,
+    x: 8,
+    q: 10,
+    z: 10
+  }
+  if (
+    myWord === '' ||
+    myWord === '\t\n' ||
```

Comment:
That one was tricky ('\t\n') - it is a string, and it has a length of two characters, yet it is just whitespace. Your line 32 seems like a good way to pass the related jasmine test! Still,  I'm curious to know whether your if statement here would also filter our longer ''empty'' strings.